### PR TITLE
 Update mapped_locations.yml with London borough mappings

### PR DESCRIPTION
**Pull request by bencmitchell**
Updating the mappings for London boroughs so that suggestions put forward by autocomplete are mapped to a polygon.

## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-1645

## Changes in this PR:

Updated mappings according to the patterns set out in the ticket, so that autocompleted locations are mapped to borough polygons.

### CODE REVIEW

File: lib/tasks/data/mapped_locations.yml [hunk 1]
```diff
@@ -2,18 +2,69 @@
 - ['barking', 'barking and dagenham']
 - ['bedfordshire', 'central bedfordshire']
 - ['brighton', 'brighton and hove']
+- ['bromley. london'. 'bromley']
```
Comment (cpjmcquillan) [inline lib/tasks/data/mapped_locations.yml:5]: typo

File: lib/tasks/data/mapped_locations.yml [hunk 2]
```diff
@@ -2,18 +2,69 @@
...
 - ['burton', 'burton upon trent']
 - ['burton-on-trent', 'burton upon trent']
 - ['burton on trent', 'burton upon trent']
+- ['camden, london', 'camden']
+- ['camden town, London', 'camden']
```
Comment (cpjmcquillan) [inline lib/tasks/data/mapped_locations.yml:10]: We're lowercase comparing everything to `London` doesn't need a capital

**Comment by david-mears-dfe** [comment]
The ticket mentions a couple of other 'royal borough of' that we need to catch, e.g. Royal Borough of Kensington and Chelsea. There are 3 in total:

https://thelondonmanwithvan.com/which-are-the-royal-boroughs-of-london/

File: lib/tasks/data/mapped_locations.yml (hunk 1 above)
Comment (bencmitchell) [reply lib/tasks/data/mapped_locations.yml]: amended, thanks

Comment (cpjmcquillan) [reply lib/tasks/data/mapped_locations.yml]: Part of it is still there lol

File: lib/tasks/data/mapped_locations.yml [hunk 3]
```diff
@@ -2,18 +2,71 @@
...
+- ['london borough of barnet', 'barnet']
+- ['london borough of bexley', 'bexley']
+- ['london borough of brent', 'brent']
+- ['london borough of bromley', 'bromley']
+- ['london borough of camden, london', 'camden']
+- ['london borough of croydon', 'croydon']
+- ['london borough of ealing', 'ealing']
+- ['london borough of enfield', 'enfield']
+- ['london borough of greenwich', 'greenwich']
+- ['london borough of hackney, london' 'hackney']
```
Comment (cpjmcquillan) [inline lib/tasks/data/mapped_locations.yml:46]: missing comma here

**Comment by david-mears-dfe** [comment]
Clicking 'details' on the failed test, I can see this error:

```
Psych::SyntaxError: (/home/runner/work/teaching-vacancies/teaching-vacancies/lib/tasks/data/mapped_locations.yml): did not find expected ',' or ']' while parsing a flow sequence at line 64 column 3
```

File: lib/tasks/data/mapped_locations.yml [hunk 4]
```diff
@@ -2,18 +2,71 @@
...
+- ['london borough of kingston upon thames', 'kingston upon thames']
+- ['london borough of lambeth, london', 'lambeth']
+- ['london borough of lewisham', 'lewisham']
+- ['london borough of merton', 'merton']
+- ['london borough of newham', 'newham']
+- ['london borough of redbridge', 'redbridge']
+- ['london borough of richmond upon thames', 'richmond upon thames']
+- ['london borough of southwark', 'southwark']
+- ['london borough of sutton', 'sutton']
+- ['london borough of tower hamlets', 'tower hamlets]
```
Comment (david-mears-dfe) [inline lib/tasks/data/mapped_locations.yml:64]: Here it is: the closing quote is missing.

Comment (bencmitchell) [reply lib/tasks/data/mapped_locations.yml]: Cheers, only just twigged that the automatic reviews ran every time I made a change and I could have used those to spot the missing punctuation :l

**Review by cpjmcquillan — APPROVED** [review]

File: lib/tasks/data/mapped_locations.yml (hunk 4 above)
Comment (david-mears-dfe) [reply lib/tasks/data/mapped_locations.yml]: Now you know!

**Pull request merged**